include/toolchain/gcc.h

```diff
@@ -65,7 +65,7 @@
 #include <toolchain/common.h>
 #include <stdbool.h>
 
-#define ALIAS_OF(of) __attribute__((alias(#of)))
+#define ALIAS_OF(of) __attribute__((weak, alias(#of)))
```

Comment:
What's the goal here?  Making all aliases weak isn't really the intent of this API (which is really just a wrapper for gcc's alias attribute).  Seems like it would be clearer to add the weak everywhere it's needed explicitly rather than change API semantics.

Reply:
Idea was to allow user to override either _open or open function with the same end result.
Agree with your point. It doesn't make sense to change API based on it's usage. Removed